python_pipeline/run_swagger_generation.py

```
@@ -19,6 +19,163 @@ def should_process_directory(dir_path: str) -> bool:
     path_parts = dir_path.split(os.sep)
     return not any(part in config.ignored_dirs for part in path_parts)
 
+
+def _api_index_output_path(directory_path: str) -> str:
```

Ruff (0.15.1)

[warning] Unused function argument: `directory_path`

(ARG001)